crates/ruff_linter/resources/test/fixtures/airflow/AIR302.py

```diff
@@ -198,28 +198,32 @@
 from airflow.operators.sqlite_operator import SqliteOperator
 from airflow.operators.trigger_dagrun import TriggerDagRunOperator
 from airflow.operators.weekday import BranchDayOfWeekOperator
+<<<<<<< HEAD
```

Comment:
Merge conflict! Do we not have any merge conflict checks in our pre-commit hook? There should be a builtin one.